### DISCUSSION TITLE
状态

### DISCUSSION THREAD

**Discussion by svefnz** [community/community#33480] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.78.0 (76820214)
Device: iPhone Xs (iOS 14.6)
</details>
![image](https://user-images.githubusercontent.com/52125806/190990095-7f06e9f5-02e8-4ec9-bca1-663c240a84ea.png)
状态栏文字居中是不是改为左对齐好一点？

**Comment by eliperkins** [comment, marked as answer]
Hey @MercuryLoveShapa, thanks for this feedback! Forgive me, since my native language is English, so I'll use Google Translate to help out here.

> 状态栏文字居中是不是改为左对齐好一点？
> Is it better to align the text in the status bar to the left?

Based on [your screenshot](https://user-images.githubusercontent.com/52125806/190990095-7f06e9f5-02e8-4ec9-bca1-663c240a84ea.png), I would 💯 agree! It looks like this is a bug, perhaps related to right-to-left languages. Thanks for passing it along! I've created an issue in our internal tracking repo to fix this. I'll let you know when this is fixed!

**Reply by svefnz** [reply]
Thank you very much for your reply！ I'm also very happy to put forward suggestions for improving the experience.
